templates/contact.html

```diff
@@ -77,7 +77,7 @@ <h4 class="text-white display-5">
       </div>
       <div class="row mt-3 mr-5">
         <h4 class="text-primary">{{ _('Report a security issue.') }}</h4>
-        <p class="text-tpo">{{ _("If you've found a security issue in one of our projects or in our infrastructure, please email tor-security@lists.torproject.org. If you want to encrypt your mail, you can get the GPG public key for the list by contacting tor-security-sendkey@lists.torproject.org or from pool.sks-keyservers.net. Here is the fingerprint:") }}</p>
+        <p class="text-tpo">{{ _("If you've found a security issue in one of our projects or in our infrastructure, please email tor-security@lists.torproject.org. If you found a bug in Tor or Tor Browser, feel free to submit a bug report for our <a href="https://hackerone.com/torproject" target="_blank">{{ _('bug bounty program.') }}</a> If you want to encrypt your mail, you can get the GPG public key for the list by contacting tor-security-sendkey@lists.torproject.org or from pool.sks-keyservers.net. Here is the fingerprint:") }}</p>
```

Comment:
Sorry, we will need to be a little more specific here:"If you've found a security bug in Tor or Tor Browser, feel free to submit it for our..."

Reply:
@gusgustavo All done :)